how-to-guides/apache-to-mongodb.md

```diff
@@ -34,7 +34,7 @@ For simplicity, this article will describe how to set up a one-node configuratio
 * [MongoDB](http://www.mongodb.org/)
 * Apache \(with the Combined Log Format\)
 
-The MongoDB Output plugin is included in the latest version of Fluentd's deb/rpm package. If you want to use RubyGems to install the plugin, please use `gem install fluent-plugin-mongo`.
+The MongoDB Output plugin is not included in the latest version of Fluentd's deb/rpm package . If you want to use RubyGems to install the plugin, please use `sudo fluent-gem install fluent-plugin-mongo`.
 
 * [Installation](../installation/)
```

Comment:
It seems weird because the installation of the plugin is described before installation of fluent-package.

It's better to follow the how-to document about kinesis:
https://github.com/fluent/fluentd-docs-gitbook/blob/0b96398486cbdbdf62117bf19185a41141a779d0/how-to-guides/kinesis-stream.md#install-kinesis-plugin